packages/block-library/src/image/edit.js

```diff
@@ -599,7 +599,7 @@ export class ImageEdit extends Component {
 							<IconButton
 								className={ classnames( 'components-icon-button components-toolbar__control', { 'is-active': this.state.isEditing } ) }
 								label={ __( 'Edit image' ) }
-								aria-pressed={ this.state.isEditing }
+								isToggled={ this.state.isEditing }
```

Comment:
Do you think `isToggled` automatically means `aria-pressed`. This was principally used for styling purposes before, so we might have to check.

Reply:
https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/button_role#Associated_ARIA_roles_states_and_properties

> Defines the button as a toggle button. The value of `aria-pressed` describes the state of the button. The values include `aria-pressed="false"` when a button is not currently pressed, `aria-pressed="true"` to indicate a button is currently pressed, and `aria-pressed="mixed"` if the button is considered to be partially pressed. If the attribute is omitted or set to its default value of `aria-pressed="undefined"`, the element does not support being pressed.

It seems like it means that but I no longer know :)

There only 2 snapshot tests failing so it looks like we rather use both.

Reply:
If this solves the same issue as #17747, I agree that it's a better direction.

Reply:
Yes, the other one is a temporary solution to unblock the next release of mobile apps.

Reply:
at least it could be marked as unstable prop with a comment to remove it.

Reply:
It's listed here for quite some time:

https://github.com/WordPress/gutenberg/tree/master/packages/components/src/button#props

Reply:
sorry I was talking about the prop added in the other PR :)

Reply:
@gziolo wouldn't "toggled" in english mean "switched from one state to the other", it's just the action of switching afaik, so I think what you really mean is here is `active`/`pressed`.

Reply:
https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/button_role

> A toggle button typically has two states: pressed and not pressed. A third mixed state is available for toggle buttons that control other elements, such as other toggle buttons or checkboxes, which do not all share the same value.  Whether an element is a toggle button or not can be indicated with the aria-pressed attribute in addition to the button role (if the element is not already a native button element):
> 
> If `aria-pressed` is not used, or is set to the "undefined" state, the button is not a toggle button.
> If `aria-pressed="false"` is used the button is a toggle button that is currently not pressed. 
> If `aria-pressed="true"` is used the button is a toggle button that is currently pressed.
> if `aria-pressed="mixed"` is used, the button is considered to be partially pressed.

It probably should be `isPressed` in that case.